homeassistant/components/filter/sensor.py

```diff
@@ -488,7 +488,7 @@ class RangeFilter(Filter, SensorEntity):
     def __init__(
         self,
         entity,
-        precision: int | None = DEFAULT_PRECISION,
+        precision: int | None = None,
```

Comment:
The `DEFAULT_PRECISION` value is still being used by the `PLATFORM_SCHEMA` definition above, so any filters that _don't_ define a precision are going to be getting the default precision passed in, which is probably why the changes below to ignore the passed in precision argument were required.  

I think if you just change `DEFAULT_PRECISION` to `None`, or potentially remove the default from the schema, the result will be the same.

Reply:
thanks por pointing that out :) refactored accordingly